source/guide_healthchecks.rst

```diff
@@ -0,0 +1,742 @@
+password, you'll need it later.
+
+::
+
+ [isabell@stardust ~]$ uberspace mail user add healthchecks
+ Enter a password for the mailbox:
+ Please confirm your password:
+ New mailbox created for user: 'healthchecks', it will be live in a few minutes...
+ [isabell@stardust ~]$
+
+Third of all your URL needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+
+We're using :manual:`Python <lang-python>` in the currently latest version 3.11:
+
+::
+
+ [isabell@stardust ~]$ python3.11 --version
+ Python 3.11.7
+ [isabell@stardust ~]$
+
+Installation
+============
+
+Download the source
+-------------------
+
+In the following steps we'll create a separate folder for this app and future apps, download the :manual:`Python <lang-python>` source code
+from GitHub and select the latest version using git tags:
+
+.. code-block:: console
+ :emphasize-lines: 21
+
+ [isabell@stardust ~]$ mkdir ~/apps
+ [isabell@stardust ~]$ cd ~/apps
+ [isabell@stardust ~]$ git clone https://github.com/healthchecks/healthchecks.git
+ Cloning into 'healthchecks'...
+ remote: Enumerating objects: 28393, done.
+ remote: Counting objects: 100% (3677/3677), done.
+ remote: Compressing objects: 100% (448/448), done.
+ remote: Total 28393 (delta 3420), reused 3349 (delta 3229), pack-reused 24716
+ Receiving objects: 100% (28393/28393), 22.57 MiB | 14.09 MiB/s, done.
+ Resolving deltas: 100% (22120/22120), done.
+ [isabell@stardust ~]$ cd healthchecks
+ [isabell@stardust ~]$ git tag
+ 1.0.2
+ v1.0
+ v1.0.1
+ v1.0.2
+ [...]
+ v2.9.2
+ v3.0
+ v3.0.1
+ v3.2
+ v3.1
+ [isabell@stardust ~]$ git checkout v3.2
+ Note: switching to 'v3.2'.
+
+ You are in 'detached HEAD' state. You can look around, make experimental
+ changes and commit them, and you can discard any commits you make in this
+ state without impacting any branches by switching back to a branch.
+
+ If you want to create a new branch to retain commits you create, you may
+ do so (now or later) by using -c with the switch command. Example:
+
+   git switch -c <new-branch-name>
+
+ Or undo this operation with:
+
+   git switch -
+
+ Turn off this advice by setting config variable advice.detachedHead to false
+
+ HEAD is now at c99b644a Update CHANGELOG for v3.2 release
+ [isabell@stardust ~]$
+
+The note of the ``git checkout`` command can be ignored as we do not plan to do any commits in this git repository.
+
+Set up the virtual environment
+-----------------
+
+Next we will setup a :manual:`Python <lang-python>` virtual environment where all dependencies are encapsulated. Then we
+install the required dependencies and on top of that also `gunicorn <https://gunicorn.org/>`_ which will be our HTTP server
+later.
+
+::
+
+ [isabell@stardust ~]$ python3.11 -m venv venv
+ [isabell@stardust ~]$ source venv/bin/activate
+ (venv) [isabell@stardust ~]$ pip3.11 install -r requirements.txt
+ Collecting aiosmtpd==1.4.4.post2 (from -r requirements.txt (line 1))
```

Comment:
Do you want me to keep all the log output from that command? Or should I somehow abbreviate it?

Reply:
That's pretty long, please abbreviate it like in the [Kimai Guide](https://lab.uberspace.de/guide_kimai/).